### ISSUE TITLE
Enrolling admin no response

### ISSUE THREAD

**Issue by yyyfor** [IBM-Blockchain-Archive/marbles#253]
I followed every instructions and successfully deployed the Marbles. 

when i connect to localhost and do all the 5 steps. when i doing the step 2, my website get stuck in this page and click try again with no response.

![image](https://user-images.githubusercontent.com/9663620/43999824-d245f1d6-9e46-11e8-89dc-ea0e6e6b1202.png)

And the logs:
```
-------------------------------- Incoming WS Msg --------------------------------
debug: [ws] received ws msg: {"type":"ping"}
debug: [fcw] Querying Channel Stats
debug: [fcw] Querying Channel Stats
debug: [fcw] Querying Channel Stats
info: [fcw] Going to enroll peer_urls=[grpc://localhost:7051], channel_id=mychannel, uuid=marblesDockerComposeNetworkmychannelOrg1MSPfabricpeerorg1, ca_url=http://localhost:7054, orderer_url=grpc://localhost:7050, enroll_id=admin, enroll_secret=adminpw, msp_id=Org1MSP, kvs_path=Users/Ming/Desktop/fabric/marbles/config//Users/Ming/.hfc-key-store/
info: [fcw] Successfully loaded enrollment from persistence
debug: added peer grpc://localhost:7051
debug: [fcw] Successfully got enrollment marblesDockerComposeNetworkmychannelOrg1MSPfabricpeerorg1
debug: [fcw] Querying Channel Stats
debug: [fcw] Querying Channel Stats
debug: [fcw] Querying Channel Stats
debug: [fcw] Querying Channel Stats
debug: [fcw] Querying Channel Stats

-------------------------------- Incoming WS Msg --------------------------------
debug: [ws] received ws msg: {"type":"ping"}
debug: [fcw] Querying Channel Stats
```

Here is the config.
```
"certificateAuthorities": {
        "fabric-ca": {
            "url": "http://localhost:7054",
            "httpOptions": {
                "verify": true
            },
            "registrar": [
                {
                    "enrollId": "admin",
                    "enrollSecret": "adminpw"
                }
            ],
            "caName": null
        }
    }
```

Please help me out. Thanks

**Comment by dshuffma-ibm** [comment]
That screen shot does not look right... What browser are you using?

Try using chrome or firefox. Clear your browser's cache, kill the app and try launching it again.  Incognito or private mode may help too.

**Comment by Abhilashrangu8** [comment]
Hello, i am facing the similar issue , did it got resolved ?

**Comment by Abhilashrangu8** [comment]
I am facing the similar issue , i am using google chrome below is the screenshot of the issue .

also i am setting it up on my local 

my marbles_local.json 

{
    "cred_filename": "connection_profile_local.json",
    "use_events": true,
    "keep_alive_secs": 120,
    "company": "United Marbles",
    "usernames": [
        "amy",
        "alice",
        "ava"
    ],
    "port": 3001
}


and 
connection_profile_local.json 
"certificateAuthorities": {
		"fabric-ca": {
			"url": "http://localhost:7054",
			"httpOptions": {
				"verify": true
			},
			"registrar": [
				{
					"enrollId": "admin",
					"enrollSecret": "adminpw"
				}
			],
			"caName": null



![screenshot from 2018-08-30 15 18 41](https://user-images.githubusercontent.com/11429749/44844329-358fe880-ac68-11e8-9df2-2a36b26816cd.png)




 could you direct me ?

**Comment by miku2000** [comment]
I had a similar problem, and the same screen (macOS, Safari). I troubleshooted (my) underlying problem as described in https://github.com/IBM-Blockchain/marbles/issues/208 , but **attention**: do not try again in the modal view ("hoveing popup") which is shown in your screenshot! Instead, close the modal view by clicking the "x". For me, **after** I fixed the underlying issues (see 208) and **after** I reattempted local start with gulp, Marbles was up and running when I closed the modal view. In particular, do not simply reload the site with the modal view in the brower: open a new tab/windows.
